src/gh/components/DF_vizualization_settings/code.py

```diff
@@ -0,0 +1,60 @@
+#! python3
+
+import System
+import typing
+
+import Rhino
+import Rhino.Geometry as rg
+from ghpythonlib.componentbase import executingcomponent as component
+
+import Grasshopper as gh
+from Grasshopper.Kernel import GH_RuntimeMessageLevel as RML
+
+from diffCheck import df_vizualization
+
+class VisualizationSettings(component):
+    def RunScript(self,
+        i_value_type,
+        i_upper_threshold,
+        i_lower_threshold,
+        i_palette,
+        i_legend_height,
+        i_legend_width,
+        i_legend_plane,
+        i_histogram_scale_factor):
+
+        """
+        Compiles all the vizualization settings to feed to the vizualization component
```

Comment:
missing type hints + docstrings. Especially for this block of settings would be great to have it 👍

Reply:
- [X] Added!